sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2192,11 +2197,12 @@ class CloudVmRayResourceHandle(backends.backend.ResourceHandle):
     - (optional) Docker user name
     - (optional) If TPU(s) are managed, a path to a deletion script.
     """
-    _VERSION = 5
+    _VERSION = 6
 
     def __init__(self,
                  *,
                  cluster_name: str,
+                 cluster_name_on_cloud: str,
```

Comment:
```suggestion
                 cluster_name_on_cloud: Optional[str],
```

Reply:
The argument should not be None. The member variable will only be None for the clusters created before this PR.